pkg/arguments/normalize.go

```diff
@@ -0,0 +1,17 @@
+package arguments
+
+import "github.com/spf13/pflag"
+
+const (
+	deprecatedInstallerRoleArnFlag = "installer-role-arn"
+	newInstallerRoleArnFlag        = "role-arn"
+)
+
+func NormalizeFlags(f *pflag.FlagSet, name string) pflag.NormalizedName {
+	switch name {
+	case deprecatedInstallerRoleArnFlag:
+		name = newInstallerRoleArnFlag
+		break
+	}
+	return pflag.NormalizedName(name)
```

Comment:
@ciaranRoche I'm wondering if we want to print a warning to the user that they're using a deprecated flag and that it will be removed at some point in the future so we can try and alter user behaviour?

I like this approach as it's seamless to the user, but it moves the burden onto us to have a plan to remove this at some point because otherwise this lives forever :-) 

Maybe we have a n+2 deprecation policy (or something) so for example:

This changes goes out in 1.2.32, therefore we expect to remove the flag in 1.2.34? For now we could simply create a JIRA ticket for 1.2.34 release so we don't forget, but in the future we could do something a bit more intelligent e.g the CI pipeline checks for flags that are beyond their allowed lifespan.

WDYT?

Reply:
@ciaranRoche Bump. How do you want to proceed with this one?

Reply:
I like the idea of the warning, right now we dont have a plan or policy on deprecating flags. I will draft a small DDR for this, so we can implement the policy and ensure its properly communicated. I worry just adding the warning now, will come back to bite us until we get acceptance from BU on it.

Reply:
If you are happy with that, I am happy for us to merge this :crossed_fingers:

Reply:
@ciaranRoche Agreed on the DDR and +1 to having agreement with the BU on a flag deprecation policy as we don't want to surprise anyone. Let me know if you want any input on the DDR.

I'll lgtm this.

cheers,

Rob